packages/camera/camera_web/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
```

Comment:
Please bump this to something >= 0.1.0, otherwise pub.dev will deduct points for having a "too low" version number.

(I normally start at anything random between 0.3.0 and 0.7.0, depending on how confident I am in the tests I have :P)

Reply:
Good to know that, thanks